app/partial/spatial/reportsPanel/reportForm/vesselFieldset/vesselFieldset.js

```diff
@@ -68,9 +68,9 @@ angular.module('unionvmsWeb').controller('VesselfieldsetCtrl',function($scope, l
                         var item = $scope.shared.vessels[idx];
                         item.type = $scope.shared.vesselSearchBy;
                         if (item.type === 'asset'){
+                            item.guid = $scope.shared.vessels[idx].historyId;

```

Comment:
Should it be 
                            item.guid = $scope.shared.vessels[idx].historyId;
or 
                            item.id = $scope.shared.vessels[idx].historyId;